qmail-start.c

```diff
@@ -3,6 +3,7 @@
 #include "exit.h"
 #include "fork.h"
 #include "auto_uids.h"
+#include "auto_users.h"
 
```

Comment:
#include "uidgid.h"

Reply:
Fixed here too, including modifying Makefile to account for the missing uidgid.h.